docs/DEVELOPING.md

```diff
@@ -343,6 +343,138 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+## Jira Connector
+
+The [Elastic Jira connector](../connectors/sources/jira.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.8_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Jira details](#gather-jira-details)
+2. [Configure Jira connector](#configure-jira-connector)
+
+#### Gather Jira details
+
+Collect the information that is required to connect to your Jira instance:
+
+- The server host url where the Jira is hosted.
+- The username the connector will use to log in to the Jira server or the service account for the Jira cloud.
+- The password the connector will use to log in to the Jira server or the API key for the Jira cloud.
+- SSL certificate if you want to establish secured connections.
+
+#### Configure Jira connector
```

Comment:
Updated the TDD to reflect the latest update, that Project Keys should be a configurable parameter. AFAIK we can update the doc right away and add it in an enhancement.

Reply:
Sure, I will update the TDD and doc with accommodating a new configuration parameter `projects`.

Reply:
Thank you!